### ISSUE TITLE
MinMax Wrapper

### ISSUE THREAD

**Issue by dylan906** [dylan906/clockpunch#4]
Add wrapper to allow `MinMax` scaling of observation states. Use sklearn class ([`MinMaxScaler`](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html)).

- Scale each observation state within itself, not whole input layer (e.g. `cov` must be scaled within itself, `obs_left` must be scaled within itself-- no mixing)
- Make sure visibility mask is not impacted.
